src/Bundle/DependencyInjection/Compiler/RegisterFiltersPass.php

```diff
@@ -44,6 +44,10 @@ public function process(ContainerBuilder $container): void
             }
 
             foreach ($attributes as $attribute) {
+                if (isset($attribute['type'], $attribute['form_type'])) {
+                    return;
```

Comment:
Should we continue with the rest of attributes, even if current one is already registered?
```suggestion
                    continue;
```

Reply:
Explanation here :

https://github.com/Sylius/SyliusGridBundle/pull/348#discussion_r1810457553

Reply:
Apparently this now works, tests are ok